test/lib/misc-junk.test.js

```diff
@@ -0,0 +1,71 @@
+const test = require('tape')
+const { getHeads } = require('../../lib/misc-junk')
+
+test('getHeads: Small graph with one head', t => {
+  //    A   (root)
+  //    |
+  //    B
+
+  const inputMap = {
+    A: { B: 1 }
+  }
+
+  const expectedHeads = ['B']
+
+  t.deepEqual(getHeads(inputMap), expectedHeads, 'one head')
+
+  t.end()
+})
+
+test('getHeads: Root-only graph', t => {
+  //    A   (root)
+
+  const inputMap = {
+    A: {}
```

Comment:
Map() never outputs this atm but it's my proposal for how it would look after https://github.com/ssbc/ssb-tangle/issues/5